src/Command/PullRequest/PullRequestCreateCommand.php

```diff
@@ -79,6 +79,7 @@ protected function configure()
 your local <comment>.gush.yml</comment> file like:
 <comment>
 table-pr:
+    branch: ['Branch', master]
```

Comment:
I know it's a command help, but IMO it's better to enforce this row to be present.